pkg/csaf/csaf_test.go

```diff
@@ -26,6 +26,19 @@ func TestOpen(t *testing.T) {
 	require.Equal(t, doc.Vulnerabilities[0].ProductStatus["known_not_affected"][0], "CSAFPID-0001")
 }
 
+func TestOpenRHAdvisory(t *testing.T) {
+	doc, err := Open("testdata/rhsa-2020_1358.json")
+	require.NoError(t, err)
+	require.NotNil(t, doc)
+	require.Equal(t, "Red Hat Security Advisory: virt:rhel security and bug fix update", doc.Document.Title)
+	require.Equal(t, "AppStream-8.1.0.Z.MAIN.EUS", doc.FirstProductName())
+
+	require.Equal(t, "Important", doc.Document.AggregateSeverity.Text)
```

Comment:
Another example, the text in the aggregate severity. Not something I expect this library to provide.